libnetwork/drivers/bridge/bridge_linux.go

```diff
@@ -908,13 +911,18 @@ func (d *driver) deleteNetwork(nid string) error {
 	return d.storeDelete(config)
 }
 
-func addToBridge(nlh *netlink.Handle, ifaceName, bridgeName string) error {
+func addToBridge(ctx context.Context, nlh *netlink.Handle, ifaceName, bridgeName string) error {
+	_, span := otel.Tracer("").Start(ctx, "libnetwork.drivers.bridge.addToBridge", trace.WithAttributes(
```

Comment:
```suggestion
	ctx, span := otel.Tracer("").Start(ctx, "libnetwork.drivers.bridge.addToBridge", trace.WithAttributes(
```
Pass the context associated with the span to logging calls so that logs could be associated with the correct trace span as events, when we get around to implementing that.